benchmarks/run_benchmarks.py

```diff
@@ -105,6 +105,7 @@ def create_parser():
     parser.add_argument('--annotations', default='', help='File containing annotations')
     parser.add_argument('--configs', help='comma seperate list of configurations')
     parser.add_argument('--start-date', help='graph start date')
+    parser.add_argument('--save-used-modules', default=False, action='store_true', help='Save used modules in a `UsedModules.cfg` file')
```

Comment:
I don't think this will be used widely enough to warrant it's own flag and we can get the same behavior with `--server-args --saveUsedModules`

If we decide to keep this here, we should move it below `--server-args` and add it to `runClient` as well (but I'm in favor of not adding it)